css/about.css

```diff
@@ -0,0 +1,132 @@
+:root {
+  --main-color: #f8412b;
+  --secondary-color: #4f4f4f;
+  --third-color: #424146;
+  --fourth-color: #eeeff1;
+}
+
+header {
+  display: flex;
```

Comment:
- [**OPTIONAL**] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```